Tools/TestWebKitAPI/Tests/WebKitCocoa/WKWebExtensionAPICommands.mm

```diff
@@ -43,6 +43,8 @@
     @"description": @"Test Commands",
     @"version": @"1.0",
 
+    @"permissions": @[ @"webNavigation" ],
```

Comment:
Does it really matter what permission I choose to put here, or can a I just arbitrarily choose one for the sake of the test (like I did here)?

Reply:
It can be anything that is valid really.